src/internal/ButtonBase.js

```diff
@@ -121,6 +103,8 @@ export default class ButtonBase extends Component {
   button = null;
   keyboardFocusTimeout = undefined;
 
+  focus = () => this.button.focus();
```

Comment:
I don't know if `findDOMNOde` is going to be removed or not.
I'm not sure how we are going to document that `focus` imperative method.
I don't have much opinion. I would keep it for now as people might hit the eslint warning.